### DISCUSSION TITLE
Does this work on a mid-2010 17" MBP with 0x0a29 (GT 330)

### DISCUSSION THREAD

**Discussion by html-hack** [julian-poidevin/MBPMid2010_GPUFix#142] [Q&A] [answered]
Hi, found a ref to this code to modify the kext files on MBPs that have the Nvidia GT 330M card. Does this fix work on MBP mid-2010 17" (MacBookPro6,1) Vendor10deDevice0a29 or only on certain other models? Started having kernel panics from this card in the last few months. Looking to apply this fix before I completely disable the discrete graphics card or open it up to fix any HW issues. Thanks!

**Comment by julian-poidevin** [comment, marked as answer]
Hello, I'm sorry, this won't work on MBP mid-2010 17".

**Comment by georgelau63** [comment]
Hi julian, it's quite difficult to reach you, except, hopefully here.
Was trying to click the Donate button in this app, but it says paypal does not accept donation from our country (Hong Kong SAR, China), any other means to donate for this wonderful app that saved my life from frequent panic during tight deadline job...

**Reply by julian-poidevin** [reply]
Hello @georgelau63, very thankful for reaching out to make a donation, you can try to donate on my Tipeee here ([https://en.tipeee.com/MBPMid2010_GPUFix](https://en.tipeee.com/MBPMid2010_GPUFix)). I don't know if that works from Hong Kong. Any way, appreciated the feedback. Donation is optional 😄

**Comment by georgelau63** [comment]
I think it did pass through.  I used the name of “Jorge White” and it’s
Euro5 dollar.

Thanks again for your work it saved my days in prime time for many many
hours…. And until now, it buys me time to change my aged MacBookPro.

From:  Julian Poidevin ***@***.***>
Reply-To:  julian-poidevin/MBPMid2010_GPUFix
***@***.***>
Date:  Thursday, 18 March 2021 at 5:33 AM
To:  julian-poidevin/MBPMid2010_GPUFix
***@***.***>
Cc:  my gmail ***@***.***>, Mention ***@***.***>
Subject:  Re: [julian-poidevin/MBPMid2010_GPUFix] Does this work on a
mid-2010 17" MBP with 0x0a29 (GT 330) (#142)


Hello @georgelau63 <https://github.com/georgelau63> , very thankful for
reaching out to make a donation, you can try to donate on my Tipeee here
(https://en.tipeee.com/MBPMid2010_GPUFix). I don't know if that works from
Hong Kong. Any way, appreciated the feedback. Donation is optional 😄

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/julian-poidevin/MBPMid2010_GPUFix/discussions/142#discus
sioncomment-495849> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ALCSJHS37HQQMVG7P7JHHR3TE
EN3ZANCNFSM4Y7GLYFA> .
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage",
"potentialAction": { ***@***.***": "ViewAction", "target":
"https://github.com/julian-poidevin/MBPMid2010_GPUFix/discussions/142#discus
sioncomment-495849", "url":
"https://github.com/julian-poidevin/MBPMid2010_GPUFix/discussions/142#discus
sioncomment-495849", "name": "View Discussion" }, "description": "View this
Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name":
"GitHub", "url": "https://github.com" } } ]

**Reply by julian-poidevin** [reply]
Yes it did pass through ! Thanks a lot for your kind donation, very glad this fix helped you. Stay safe.

**Comment by georgelau63** [comment]
Dear Julian,

A quick question tho.  Is it safe to update future OS Update, and upon
finish to rerun your fix again due to new OS Update may reset things.

Is it safe?

Tks
George

From:  Julian Poidevin ***@***.***>
Reply-To:  julian-poidevin/MBPMid2010_GPUFix
***@***.***>
Date:  Thursday, 18 March 2021 at 3:39 PM
To:  julian-poidevin/MBPMid2010_GPUFix
***@***.***>
Cc:  george HKDPL ***@***.***>, Mention
***@***.***>
Subject:  Re: [julian-poidevin/MBPMid2010_GPUFix] Does this work on a
mid-2010 17" MBP with 0x0a29 (GT 330) (#142)


Yes it did pass through ! Thanks a lot for your kind donation, very glad
this fix helped you. Stay safe.


You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/julian-poidevin/MBPMid2010_GPUFix/discussions/142#discus
sioncomment-496962> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ALCSJHQL56ZXZ3CE4H6QPRDTE
GU2FANCNFSM4Y7GLYFA> .
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage",
"potentialAction": { ***@***.***": "ViewAction", "target":
"https://github.com/julian-poidevin/MBPMid2010_GPUFix/discussions/142#discus
sioncomment-496962", "url":
"https://github.com/julian-poidevin/MBPMid2010_GPUFix/discussions/142#discus
sioncomment-496962", "name": "View Discussion" }, "description": "View this
Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name":
"GitHub", "url": "https://github.com" } } ]

**Comment by julian-poidevin** [comment]
@georgelau63 : the app is not compatible with the latest macOS version, I'd recommend not to update.

**Reply by simsimlinlin** [reply]
Is there any other recommendation for big sur for example? 
many thanks in advance

**Reply by julian-poidevin** [reply]
@simsimlinlin sorry, I don't have any recommandations

**Comment by georgelau63** [comment]
Thanks for the tip!

「Julian Poidevin ***@***.***>」在 2021年5月1日週六，下午5:59 寫道：
